main.py

```diff
@@ -0,0 +1,113 @@
+import sqlite3
+import random
+import threading
+import os
+import dbActions
+import Settings
+
+
+def finish_game(name, level):
+    """
+    inserts player's name and score, and finishes the game.
+    :param name: player's name
+    :param level: player's score
+    """
+    con = raise_database()
+    dbActions.insert_new_score(name, level, con, Settings.db_table_name)
+    print "game over"
+    os._exit(1)
```

Comment:
Look at [sys.exit vs os._exit](http://stackoverflow.com/questions/9591350/what-is-difference-between-sys-exit0-and-os-exit0)